modules/ols-large-language-model-configuration-overview.adoc

```diff
@@ -15,7 +15,7 @@ For more information, see the official {openai} link:https://platform.openai.com
 
 == About Azure OpenAI configuration with OpenShift Lightspeed
 
-You must have an existing deployment of {azure-openai}, and you must know which subscription the {azure-openai} deployment is associated with.
+You need a {azure-openai} service instance. There must be at least one model deployment in {azure-studio} for that instance.
```

Comment:
You need *an* (I think)

Reply:
If it were Azure OpenAI, yes. It actually renders as Microsoft Azure OpenAI, so it should be a, but I see why you thought that looking at the code.